packages/backend/src/graphql/resolvers/accounting_transfer.tigerbeetle.test.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ * @jest-environment ./packages/backend/jest.tigerbeetle-environment.ts
+ */
+
+import { IocContract } from '@adonisjs/fold'
```

Comment:
I think we could probably get rid of this test file and just add tests for `getAccountTransfers` in `/src/accounting/tigerbeetle/service.test.ts`, similar to how you did for the PSQL tests. 
IMO resolver tests shouldn't really care what is used under the hood, just that they return the correct response and possible errors

Reply:
I picked up an issue during the testing that only occurred in the resolver. So I added it due to test coverage and to prevent it from creeping up in the future. If it is ok, I would like to leave it?